packages/ibm-products/src/components/AboutModal/AboutModal.stories.js

```diff
@@ -160,49 +166,60 @@ const commonArgs = {
           { fontWeight: '600' }
         }
       >
-        Watson AI Ops
+        Product name
       </span>
     </>
   ),
-  content: <>This is example content for an About Modal.</>,
+  versionNumber: <>Version 0.0.0</>,
+  copyrightText: <>Copyright &copy; IBM Corp. 2020, 2023</>,
 };
 
 const aboutModalStoryName = 'About modal';
 export const aboutModal = prepareStory(
   Template.bind({}, aboutModalStoryName, true),
   {
     storyName: aboutModalStoryName,
+    args: {
+      additionalInfo: 0,
+      ...commonArgs,
+    },
+  }
+);
+
+const withAdditionalInfoStoryName = 'About modal with additional info (footer)';
+export const withAdditionalInfo = prepareStory(
+  Template.bind({}, withAdditionalInfoStoryName, true),
+  {
+    storyName: withAdditionalInfoStoryName,
     args: {
       additionalInfo: 1,
       ...commonArgs,
     },
   }
 );
 
-const withAllBodyStoryName =
-  'About modal with links and legal and copyright text';
-export const withAllBody = prepareStory(
-  Template.bind({}, withAllBodyStoryName, true),
+const withLinksStoryName = 'About modal with links';
```

Comment:
We can actually probably remove the `with links` story since the `links` prop in Storybook will allow you to add/remove links to the story.

In fact, most things you should be able to accomplish in a 1-2 stories if you update the controls to be more like links. But anything that would be a bit more involved to update (e.g. different variations of `content`) **can definitely be addressed later** than just removing redundant stories like links or even “all props set.”